openconfig/featureprofiles
# default static route test

static route test with the following steps

- Test case steps:
1.  Configures DUT interfaces port1 and port2 with VRF name VRF1
2. Configure DUT Interface with IPv4 and IPv6 Address
4. Configures OTG interfaces and creates 
5. Create Traffic flow with Src address of OTG port 1 interface and dst address of 203.0.113.x
6. Configure default static route under VRF VRF1 
7. Verify traffic flow has no loss
